guide/popular-topics/common-questions.md

```diff
@@ -59,6 +59,28 @@ if (<message>.author.id === '<id>') {
 }
 ```
 
+### How do I check which role was added/removed, and for which member?
+
+<!-- eslint-skip -->
+
+```js
+// We start by declaring a guildMemberUpdate listener
+// This should ***not*** be inside the message listener (or any other listener)
+
+<client>.on('guildMemberUpdate', (oldMember, newMember) => {
+	if(oldMember.roles.size === newMember.roles.size) return;
```

Comment:
This suggests that roles can only be changed if the size of the role collection changes, which is wrong.
Roles can not only be added or removed, and not only one by one either (relevant further down for the usage of `find`). 

The approach I tend to suggest to people with this request is the following:
is in old but not in new => removed
is in new but not in old =>added
(is in both => unchanged) which i'd not include here